package/MDAnalysis/analysis/distances.py

```diff
@@ -55,6 +54,13 @@ def contact_matrix(coord, cutoff=15.0, returntype="numpy", box=None):
     .. versionchanged:: 0.11.0
        Keyword *suppress_progmet* and *progress_meter_freq* were removed.
     '''
+    try:
```

Comment:
This can be put in the sparse return branch, and it needs to raise a proper ImportError rather than returning nothing